client/src/DisplayPlayers.js

```diff
@@ -0,0 +1,32 @@
+import React, {useState} from 'react';
+import {useInput} from "./playerReacter";
+
+export const increaseStat = value => {
+    return value + 1;
+}
+
+const DisplayPlayers = (props) => {
```

Comment:
I'd recommend destructing your props so that instead of
```
const DisplayPlayers = (props) => {
...
            {props.players.map((player) =>
...
}
```
you could do
```
const DisplayPlayers = ({ players }) => {
...
            {players.map((player) =>
...
}
```

to keep it DRY 🏜